### ISSUE TITLE
Interface: Data reports return no values.

### ISSUE THREAD

**Issue by roger-mahler** [humlab/sead_clearinghouse#24]
Click on any report produces the following client side error:
```
Deprecated: Function create_function() is deprecated in /home/clearinghouse/public/api/InfraStructure/Utility.php on line 56
{"data":[],"columns":[{"column_name":"Sample Name","column_field":"sample_name","data_type":"text"}],"options":{"paginate":true}}
```

**Comment by roger-mahler** [comment]
Cause: Use of deprecated `create_function`in PHP 7.2. [stackoverflow](https://stackoverflow.com/questions/48161526/php-7-2-function-create-function-is-deprecated)

**Comment by roger-mahler** [comment]
Was also resolved by #22.
